library/Test/HSpec/JUnit.hs

```diff
@@ -1,115 +1,123 @@
 module Test.HSpec.JUnit
-  ( runJUnitSpec
-  , configWith
```

Comment:
Is it possible to keep these two functions, at their present types, defined in terms of `junitFormat`?

If so, we could make this addition as minor-only and still remove all the code and have a clean implementation. If no, oh well.

Reply:
Yeah, they could be kept.